cmd/dev_common.go

```diff
@@ -379,6 +379,11 @@ func commonCmd(config *devCommonConfig, mode string) error {
 			return portsErr
 		}
 
+		debugPort, debugPortErr := GetEnvVar("APPSODY_DEBUG_PORT", config.RootCommandConfig)
```

Comment:
If we want to support multiple debug ports, we will need to split the value of the ENV here

Reply:
We will also need to sanitize and verify these values